root/etc/s6-overlay/s6-rc.d/init-endlessh-config/run

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/with-contenv bash
+
+mkdir -p /config/logs/endlessh
```

Comment:
this line can probably be removed as it's duplicated right after

Reply:
Oops. Was because when I first added it I hadn't sorted the logging properly, and it was breaking if LOGFILE=false.